public/index.html

```diff
@@ -0,0 +1,71 @@
+﻿<!doctype html>
+<html>
+<head>
+    <link href="css/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/ageg.css" rel="stylesheet" />
+
+    <!-- Include the AngularJS library -->
+    <script src="https://code.angularjs.org/1.4.4/angular.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.4/angular-route.min.js"></script>
+    <script src="js/jquery-1.11.3.min.js"></script>
```

Comment:
Please use CDNs for script files.
It's not very important now, but it's a good thing to have working before going to prod.
(have a look at the other user branch for examples...)